### PR TITLE
Feature account templates

### PR DESCRIPTION
**Pull request by andela-tadesanya**
HTML templates for user sign in, sign up and reset password are done.

### CODE REVIEW

File: troupon/account/views.py.orig [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+                )
+        else:
+            # raise 404 when the hash doesn't return a user:
+            raise Http404("/User does not exist")
+
+    def post(self, request, *args, **kwargs):
+        reset_password_form = ResetPasswordForm(request.POST,auto_id=True)
+        if reset_password_form.is_valid():
+            try:
+                # get the recovery_user from the session:
```
Comment (andela-cnnadi) [inline troupon/account/views.py.orig]: I'm not sure why this file is here...

File: troupon/templates/base.html.orig [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+<html lang="en">
+
+<head>
+
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
```
Comment (andela-cnnadi) [inline troupon/templates/base.html.orig]: Don't know why this file is here too...

File: troupon/troupon/settings/development.py.orig [hunk 3]
```diff
@@ -0,0 +1,135 @@
+"""
+Django settings for troupon project.
+
+Generated by 'django-admin startproject' using Django 1.8.3.
```
Comment (andela-cnnadi) [inline troupon/troupon/settings/development.py.orig]: I don't know why this file is here too...

Comment (andela-tadesanya) [inline troupon/troupon/settings/development.py.orig]: Will go through them and remove them asap.

**Comment by andela-tadesanya** [comment]
I have removed all unwanted files.

**Pull request merged**